reporting/docker-compose.yml

```diff
@@ -20,3 +20,18 @@ services:
     image: openlmis/postgres:${OL_POSTGRES_VERSION}
     env_file: ../settings.env
     depends_on: [consul]
+
+  zookeeper:
+    image: confluentinc/cp-zookeeper:${OL_ZOOKEEPER_VERSION}
+    environment:
+      - ZOOKEEPER_CLIENT_PORT=32181
+      - ZOOKEEPER_TICK_TIME=2000
+      - ZOOKEEPER_SYNC_LIMIT=2
+
+  kafka:
+    image: confluentinc/cp-kafka:${OL_KAFKA_VERSION}
+    environment:
+      - KAFKA_ZOOKEEPER_CONNECT=localhost:32181
```

Comment:
Localhost doesn't work here because it's relative to the docker container. I'm getting Kafka connection errors when trying to access zookeeper. Instead, we should use a .env variable.